src/Container.php

```diff
@@ -258,6 +260,12 @@ public function addWhisperer(string $name, $label = null, array $items = []): Co
 	}
 
 
+	public function addDivider(string $previousControl, Html|string $content): void
```

Comment:
$previousControl bych dal optional, a pokud nebude zadaný, tak by se to vložilo za poslední vložený input (to by mělo jít vytáhnout odněkud)